main.rb

```diff
@@ -6,57 +6,60 @@ def main
     puts welcome_screen
 
     print "Please enter the dimension of the table [default: 3 (3 means 3x3)]: "
-    dimension = gets.strip.to_i
 
-    player_name1,player1_symbol = setup_player(1)
+    dimension = gets.strip.to_i
```

Comment:
Alvaro when you say "these two", do you mean `dimension = gets.strip.to_i` and this `game = Game.new(dimension)` ? if so, maybe we can leave the initialization of the game outside of that method (I mean, in the place where it is now) because the method should to return the game and I don't see the point of that.

Reply:
It's good in its current state

Reply:
👍